grafana-dashboard.tf

```diff
@@ -0,0 +1,36 @@
+resource "random_integer" "graph_id" {
+  min     = 1000
+  max     = 999999
+  count = length(local.apiary_data_buckets)
+}
+
+data "template_file" "graphs" {
+  template = file("${path.module}/templates/graph.tpl")
+  count = length(local.apiary_data_buckets)
+  vars = {
+    bucket_name = local.apiary_data_buckets[count.index]
+    title_bucket_name = local.apiary_managed_schema_names_replaced[count.index]
+    graph_id = random_integer.graph_id.*.result[count.index]
+  }
+}
+
+data "template_file" "dashboard" {
+  template = file("${path.module}/templates/dashboards.tpl")
+  vars = {
+    panels = "[${join(",", data.template_file.graphs.*.rendered)}]"
+  }
+}
+
+resource "kubernetes_config_map" "grafana-dashboard" {
```

Comment:
Are these dashboards only supposed to be created when Apiary metastores are running in k8s?  

If not, how does it work for ECS?

If so, then you need to constrain this resource to only be used in a k8s deployment with:
     `count = "${var.hms_instance_type == "k8s" ? 1 : 0}"`

Reply:
👍  If they're using a config map, I'd assume so :)